### PR TITLE
Trev8939/scene property expressions

### PR DESCRIPTION
**Pull request by TADraeseke**
Hi @priy1289 - Can I ask you to review this new sample for me when you have a moment? Thanks!

### CODE REVIEW

File: java/scene-property-expressions/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## How it works
+
+Simple renderers can dynamically update the positions of graphics using
+an expression. The expression relates a renderer property to one of the
+graphic's attribute keys.
+
+1. Create a new `GraphicsOverlay`. 
+2. Create a new `SimpleRenderer` and set its scene properties.
+3. Set the heading expression to `[HEADING]` and the pitch expression to
```
Comment (priy1289) [inline java/scene-property-expressions/README.md]: Minor: Expressions on the app are not Capital case.

File: java/scene-property-expressions/src/main/java/com/esri/arcgisruntime/sample/scenepropertyexpressions/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      @Override public void onStartTrackingTouch(SeekBar seekBar) {
+
+      }
+
+      @Override public void onStopTrackingTouch(SeekBar seekBar) {
+
+      }
+    });
+  }
+}
```
Comment (priy1289) [inline java/scene-property-expressions/src/main/java/com/esri/arcgisruntime/sample/scenepropertyexpressions/MainActivity.java]: onPause, onResume, onDestroy lifecycle methods to be added..

File: java/scene-property-expressions/src/main/java/com/esri/arcgisruntime/sample/scenepropertyexpressions/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      @Override public void onStartTrackingTouch(SeekBar seekBar) {
+
+      }
+
+      @Override public void onStopTrackingTouch(SeekBar seekBar) {
+
+      }
+    });
+    SeekBar pitchSeekBar = findViewById(R.id.pitchSeekBar);
+    cone.getAttributes().put("PITCH", pitchSeekBar.getProgress() - 90);
```
Comment (priy1289) [inline java/scene-property-expressions/src/main/java/com/esri/arcgisruntime/sample/scenepropertyexpressions/MainActivity.java]: maybe a int constant? as its used at 3 places..

File: java/scene-property-expressions/src/main/java/com/esri/arcgisruntime/sample/scenepropertyexpressions/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+    // create a red cone graphic
+    SimpleMarkerSceneSymbol coneSymbol = SimpleMarkerSceneSymbol.createCone(Color.RED, 100, 100);
+    coneSymbol.setPitch(-90);  // correct symbol's default pitch
+    Graphic cone = new Graphic(new Point(83.9, 28.41, 200, SpatialReferences.getWgs84()), coneSymbol);
+    graphicsOverlay.getGraphics().add(cone);
+
+    // bind attribute based on values in seek bars
+    SeekBar headingSeekBar = findViewById(R.id.headingSeekBar);
+    cone.getAttributes().put("HEADING", headingSeekBar.getProgress());
```
Comment (priy1289) [inline java/scene-property-expressions/src/main/java/com/esri/arcgisruntime/sample/scenepropertyexpressions/MainActivity.java]: minor - HEADING and PITCH could be string constants.

**Comment by TADraeseke** [comment]
Thanks @priy1289 - Made those changes, merging now!

**Pull request merged**